lms/djangoapps/course_home_api/dates/v1/views.py

```diff
@@ -58,13 +60,16 @@ class DatesTabView(RetrieveAPIView):
 
     def get(self, request, *args, **kwargs):
         course_key_string = kwargs.get('course_key_string')
+        course_key = CourseKey.from_string(course_key_string)
+
+        if not course_home_mfe_dates_tab_is_active(course_key):
+            return Response(status=status.HTTP_302_FOUND)
```

Comment:
A redirect without a Location seems like surprising behavior. Especially odd on an API request. Might be better to return a 404 or some other code?